src/content/docs/en/pages/devtools/cli/azion-cli/azion-overview/azion-overview.mdx

```diff
@@ -1,10 +1,13 @@
 ---
 title: Azion CLI
-description: Dive into Azion CLI, your tool for optimized edge management. Learn commands, features, and that allow you to interact with Azion products via terminal.
-meta_tags: cli, next, javascript, astro, vue, react, edge computing
+description: 'Explore Azion CLI for edge computing: manage and deploy edge applications,
```

Comment:
It has '' in the description, different from others.

Reply:
```suggestion
description: Explore Azion CLI for edge computing: manage and deploy edge applications,
```